perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -230,6 +230,9 @@ sub check_and_stash_category : Private {
     my $all_areas = $c->stash->{all_areas};
     my @bodies = $c->model('DB::Body')->active->for_areas(keys %$all_areas)->all;
     my %bodies = map { $_->id => $_ } @bodies;
+    my @body_names_to_list = map { $_->name } values %bodies;
+    $c->{stash}->{bodies_to_list} = \%bodies;
+    $c->{stash}->{body_names_to_list} = \@body_names_to_list;
```

Comment:
The ajax call that returns the same data calls list `list_of_names`, might be good to match that here? (Not that it's a great name.)